src/test/routes/confirmDeleteItem.spec.ts

```diff
@@ -27,7 +27,7 @@ describe("Confirm Delete page", () => {
     });
 
     it("should return Confirm Delete User Profile page when authenticated", () => {
-      idamServiceMock.resolveRetrieveUserFor("1", "admin");
+      idamServiceMock.resolveRetrieveUserFor("1", "ccd-import");
```

Comment:
Wondering whether we could declare a constant for `ccd-import`

Reply:
Good idea; I'll introduce a `const` for "ccd-import"